tests/junit.py

```diff
@@ -28,13 +28,17 @@ def create_argparser() -> argparse.ArgumentParser:
     parser.add_argument('-b', '--junit-batch', type=str, default='', help='Name of JUnit batch for output file')
     parser.add_argument('-np', '--pool-size', type=int, default=1, help='Number of test cases to run in parallel')
     parser.add_argument('-s', '--smartredis_dir', type=str, default='', help='path to SmartSim library, if present')
+    parser.add_argument('--use_torch', type=bool, default=False, help='Whether to build with torch')
     parser.add_argument('test', help='Test executable', nargs='?')
 
     return parser
 
 
 # Necessary functions for running tests
 class CeedSuiteSpec(SuiteSpec):
+    def __init__(self, use_torch: bool):
+        self.use_torch: bool = use_torch
```

Comment:
Minor consistency - in Ratel we'd say `has_torch` to indicate the env has Torch available